### PR TITLE
Create Phoenix Location

### PR DESCRIPTION
**Pull request by JayCarpenter**
---
githubHandle: Jay101
latitude: 28.291956
longitude: -81.407571
---

### CODE REVIEW

File: Phoenix Location [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Hello World</h1>
```
Comment (JayCarpenter) [inline Phoenix Location]: Just practicing!

Comment (medh2000) [inline Phoenix Location]: me too

**Comment by beardofedu** [comment]
@JayCarpenter if you remove the `Phoenix Location` file it should allow the test to pass and you can merge your Pull Request. 👍

File: Phoenix Location (hunk 1 above)
Comment (calbarkers) [inline Phoenix Location]: Also practicing

Comment (Tyrudite) [inline Phoenix Location]: # Good job so far !!
_I'm pretty sure the majority of us are in our personal development stages , so just keep practicing!!_
**Don't forget to add the reference: @githubteacher 

> 

> **

**Comment by ceg3** [comment]
@JayCarpenter This looks good 🎉

**Comment by beardofedu** [comment]
@JayCarpenter I went ahead and fixed the syntax issues in `0e7c`. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 19th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**